frontend/public/components/_dropdown.scss

```diff
@@ -118,6 +115,10 @@ $dropdown-background--hover: $color-pf-black-200; // pf-c-dropdown__menu-item--h
 
 // Custom dropdown menu rules for namespace-selector & resource-item that don't use the default pf4 styles
 .co-namespace-selector .dropdown-menu__autocomplete-filter {
+  li.dropdown-menu__filter {
```

Comment:
Can you describe the reason for the CSS changes here? This change seem less BEM-y, and we try to avoid nested selectors when we can.